wpspin.html

```diff
@@ -1336,6 +1336,7 @@
 ASUS RT-N300
 ASUS RX3041 Rev.B
   04:92:26:xx:xx:xx
+  04:D9:F5:E9:Ax:xx
```

Comment:
```suggestion
  04:D9:F5:xx:xx:xx
```
Technically valid for `04:D9:F5:5*` .. `04:D9:F5:F*`